src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -1757,295 +1282,6 @@ public ImmutableArray<DocumentId> GetRelatedDocumentIds(DocumentId documentId)
                 (solution: this, projectState.Language));
         }
 
-        /// <summary>
-        /// Creates a new solution instance with all the documents specified updated to have the same specified text.
-        /// </summary>
-        public SolutionState WithDocumentText(IEnumerable<DocumentId?> documentIds, SourceText text, PreservationMode mode)
-        {
-            var solution = this;
-
-            foreach (var documentId in documentIds)
-            {
-                if (documentId == null)
-                {
-                    continue;
-                }
-
-                var doc = GetProjectState(documentId.ProjectId)?.DocumentStates.GetState(documentId);
-                if (doc != null)
-                {
-                    if (!doc.TryGetText(out var existingText) || existingText != text)
-                    {
-                        solution = solution.WithDocumentText(documentId, text, mode);
-                    }
-                }
-            }
```

Comment:
note: i smiplified this method when i moved it.  lined 1774-1777 are already handled by solution.WithDocumentText.  i will call this out at the callsite.